### ISSUE TITLE
Suggestion for additional read view flexibility

### ISSUE THREAD

**Issue by caktus-philip** [caktus/django_bread#25]
At present one can change the read view display by providing a different form class. (It defaults to a `ModelForm`.) Despite being able to change the class, there's still no way to display anything that doesn't fit neatly into a form widget.

It would be nice to be able to eschew the form entirely and provide to the view/template a list of 2-tuples of `(label, name)` where the name can be an attribute name or some kind of callable. Specifically, I suggest that the following three kinds of names would cover most use cases:
 - The name of a model attr, to be resolved as `getattr(instance, name)`
 - The name of a model method, to be resolved as `getattr(instance, name)()`
 - The name of a non-model function, to be resolved as `name(request)` or `name(context_data)`. Note that the object instance is available in the context data as `context['object']`.

This list of 2-tuples could be passed to bread at init, or perhaps as a `ReadView` attribute. It would be up to the bread view/template to recognize that the list of 2-tuples is populated and use it in favor of the form.

I wrote some experimental code inside `bread.py` as an experiment; I'll put it here in case it's useful for implementation.
```
    def _get_field_value(self, name, data):
        if isinstance(name, basestring) and hasattr(self.object, name):
            # This is an instance attr or method
            attr = getattr(self.object, name)
            value = attr() if callable(attr) else attr
        else:
            # This is a non-instance method
            if callable(name):
                value = name(data)
            else:
                name = str(name)
                raise ValueError, 'Field "%s" is not a callable or object attribute/method' % name
        return value

```

**Comment by dpoirier** [comment]
My feeling is that for any customization of the read view beyond the simplest, it'll be easier to just create a custom template.  The default template with the form was just intended as the bare minimum to get a view started.

**Comment by caktus-philip** [comment]
Largely implemented by #36
